packages/components/react/src/Panels/PreviewPanel.tsx

```diff
@@ -61,6 +61,13 @@ export const PreviewPanel = memo(
       [],
     );
 
+    // we update the iframes position at ~30fps, needed for mobile
+    useEffect(() => {
+      const intervalId = setInterval(onResize, 20);
```

Comment:
We don't want to use `requestAnimationFrame` instead? Or would that be too much?

Reply:
Good question, I'm torn on that one and I'm not sure. I feel like it would be too much, however if we did that then we can discard the `ResizeObserver` logic entirely which is also pretty neat.

Reply:
I think I would prefer/try to use `requestAnimationFrame`. We could use the callback argument to calculate it so that it will resize at approximately 60 fps if we wanted that (or 30) https://developer.mozilla.org/en-US/docs/Web/API/Window/requestAnimationFrame#examples. But if everyone is ok to just use `setInterval()`, I'm fine with it as well though.

Reply:
Yea I'd also prefer `requestAnimationFrame` so we can use the max number of frames that the browser allows us to render at.